index.html

```diff
@@ -8,24 +8,77 @@
     <!--CSS Stylesheet-->
     <link rel="stylesheet" type="text/css" href="styles.css" />
     <!--Boostrap-->
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
-        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
 </head>
 
 <body class="body">
+    <!--Header-->
     <header>
-        <div class="header">
-            <nav>
-                
-            </nav>
+        <div class="container">
+            <div class="toolbar">
+                <header>
+                    <div class="logo">
+                        <a href="#">
+                            <span class="my-logo">
+                                My Logo
+                            </span>
+                        </a>
```

Comment:
[OPTIONAL] Please align your logo with the hamburger icon like on the template.
<img width="405" alt="Screenshot 2023-01-18 at 10 23 19 PM" src="https://user-images.githubusercontent.com/25887253/213275378-099c275a-65df-4f00-9be0-986a860646fc.png">
<img width="405" alt="Screenshot 2023-01-18 at 10 24 03 PM" src="https://user-images.githubusercontent.com/25887253/213275381-8979e388-c1d5-413a-b5c3-9384ebfafb15.png">